keyboards/romac/config.h

```diff
@@ -0,0 +1,57 @@
+#ifndef CONFIG_H
```

Comment:
You can replace all your include guards with `#pragma once`

```suggestion
#pragma once
```